db/migrate/20170812152912_add_unique_constraint_to_craftsmen_employment_id.rb

```diff
@@ -0,0 +1,18 @@
+class AddUniqueConstraintToCraftsmenEmploymentId < ActiveRecord::Migration
+  def change
```

Comment:
Another note is that you could probably use activerecord to add a foreign key, then it'll take care of the naming for us.